Client.php

```diff
@@ -84,11 +84,16 @@ class Client extends Connection
      */
     public function __construct(
         Socket\Client $client,
-        $endPoint               = '/',
+        $endPoint               = null,
         Http\Response $response = null
     ) {
         parent::__construct($client);
-        $this->setEndPoint($endPoint);
+
+        $this->setEndPoint(is_null($endPoint)?'/':$endPoint);
```

Comment:
Correct. We prefer `null ==` instead of `is_null` but that's a detail.
Changing for `null` is correct, bravo.